### ISSUE TITLE
Typename checks on release

### ISSUE THREAD

**Issue by j-burgos** [ayoubdev/react-native-android-kit#13]
`if(obj.type.name !== 'TabAndroid' || obj.type.__proto__.name !== 'Tab')`

When generating a release APK these checks do not pass and the control does not behave correctly

**Comment by ayoubdev** [comment]
Could you please show some logs ? And be more accurate concerning "the control does not behave correctly" ?

**Comment by j-burgos** [comment]
Hi @ayoubdev,
Great tab control by the way.

The problem I faced, was that I created a release APK (`./gradlew assembleRelease`) and the control did not show the tabs text and neither the tab content. Those were both blank.

On the app's logs, I got a `TabLayoutAndroid View must only have TabAndroid as direct children`

I added some logs to the `containMixViews` method in your component and found out that on runtime with the release APK `obj.type.name` returned `t` and `obj.type.__proto__.name` returned just `r`

Then, I tested a debug APK (``./gradlew assembleDebug`) and there, those properties returned the correct `TabAndroid` and `Tab` respectevely.

I believe it has to do with React Native minifying code when creating a release APK and the name of the types not matching
